### DISCUSSION TITLE
A nice libray based on VUE3, Can we absorb some of their features to make NiceGUI faster?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#3355] [Ideas / Feature Requests]

**Comment by python-and-novella** [comment]


**Reply by v479038280** [reply]
Yes,It's mainly python's problem,  the interface  of 'nativeUI'  is indeed neat and tidy.

**Reply by v479038280** [reply]
I haven't done a benchmark, but intuitively, the native UI is definitely faster to load than Quasar.

**Reply by v479038280** [reply]
Sorry ,I'm still a learner ,don't know much about the thing behind nicegui,or the tech behind quasar or nativeui

**Reply by v479038280** [reply]
I like  the  responsive concept of ex4nicegui,only excute when the data changed

**Reply by v479038280** [reply]
I don't like streamlit ,it's not that flexible

**Comment by waqi786** [comment]
It's primarily an issue with Python
